ui/components/app/signature-request/signature-request.js

```diff
@@ -256,161 +269,191 @@ const SignatureRequest = ({ txData }) => {
   ///: END:ONLY_INCLUDE_IF
 
   return (
```

Comment:
We should probably get someone from the confirmations UX team to review these changes too.

And probably QA all the changed confirmations